runtime/server/instances.go

```diff
@@ -201,11 +202,19 @@ func (s *Server) GetLogs(ctx context.Context, req *runtimev1.GetLogsRequest) (*r
 		return nil, ErrForbidden
 	}
 
-	logs, err := s.runtime.InstanceLogs(ctx, req.InstanceId)
+	if req.Level == "" {
+		req.Level = "INFO"
+	}
+	lvl := toRuntimeLogLevel(req.Level)
+	if lvl == runtimev1.LogLevel_LOG_LEVEL_UNSPECIFIED {
+		return nil, status.Errorf(codes.InvalidArgument, "invalid log level %s", req.Level)
+	}
```

Comment:
After switching to `LogLevel` proto type, can replace this with a check that changes `UNSPECIFIED` to `INFO` (since `UNSPECIFIED` is the default value if the log level isn't explicitly passed, as might be the case from old clients).